.github/workflows/test-harness-acapy.yml

```diff
@@ -27,6 +27,9 @@ jobs:
   test:
     runs-on: ubuntu-latest
     if: (github.event_name == 'schedule' && github.repository == 'hyperledger/aries-agent-test-harness') || (github.event_name != 'schedule')
+    env:
+      LEDGER_URL_CONFIG: "http://localhost:9000"
+      TAILS_SERVER_URL_CONFIG: "http://localhost:6543"
```

Comment:
I don't think "localhost" will work here, it needs to be the internal IP address of the docker network (in the `manage` script we set `$DOCKERHOST`, which comes from:

`export DOCKERHOST=${APPLICATION_URL-$(docker run --rm --net=host eclipse/che-ip)}`

Reply:
my latest commit fixes this as well - `localhost` in these URLs will be replaced with `$DOCKERHOST` for the URLs passed to the agents.